public/pancakeswap-top-100.json

```diff
@@ -0,0 +1,797 @@
+{
```

Comment:
Do we need to commit this file? I don't think so since we fetch it in `useTokenList`

Reply:
I tried fetching from github. I was getting some cors errors and thought moving it to the repo would be easier with the idea of potentially moving it to IPFS if we want to remove it from our bundle. It still uses the useTokenList method and fetches the local file.

Reply:
Will a browser always load this file in the initial bundle / would it slow page load times with this approach? Regardless, we can always track removing/optimizing this in a separate issue

Reply:
Hm, I am not sure, but I will make a ticket for this optimization

Reply:
#38